calico_containers/tests/st/test_no_powerstrip.py

```diff
@@ -0,0 +1,60 @@
+from sh import ErrorReturnCode
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestNoPowerstrip(TestBase):
+    def test_no_powerstrip(self):
+
+        host = DockerHost('host')
+
+        calicoctl = "/code/dist/calicoctl %s"
+
+        host.execute("docker run --rm  -v `pwd`:/target jpetazzo/nsenter", _ok_code=[0, 1])
+
+        host.execute(calicoctl % "node --ip=127.0.0.1")
+        host.execute(calicoctl % "profile add TEST_GROUP")
+
+        self.assert_powerstrip_up(host)
+
+        # Remove the environment variable such that docker run does not utilize
+        # powerstrip.
+        host.execute("docker run -e CALICO_IP=192.168.1.1 -tid --name=node1 busybox")
```

Comment:
`docker_host=False` would be more clear here.  It's confusing when you say remove the environment variable, but have left the `-e CALICO_IP=...` in.